Cargo.toml

```diff
@@ -98,7 +97,6 @@ type-system.path                    = "libs/@blockprotocol/type-system/rust"
 
 # External dependencies
 ada-url                  = { version = "=3.2.4", default-features = false }
-ahash                    = { version = "=0.8.12", default-features = false }
```

Comment:
I accidentally ran `cargo add` on the main repo again 🙃 anyway, may be good as a drive-by to just remove these external dependencies (but keep `hash-graph-migrations`). What do you think @TimDiekmann

Reply:
Yes, let's keep the list above, but remove unused external dependencies

Reply:
thanks for telling me twice buddy 👍